coop_shift/model/res_partner.py

```diff
@@ -22,11 +21,11 @@
 ##############################################################################
```

Comment:
Please replace this big header with the new style, respecting original authors and copyrights, and adding druidoo